src/rsa.rs

```diff
@@ -127,6 +127,30 @@ fn verify(params: &RSAParameters, msg: untrusted::Input,
 }
 
 
+pub mod primitive {
+    use untrusted;
+    /// Lower-level verification function for RSA signatures.
+    ///
+    /// If you do not have the public key available as DER-encoded
+    /// PKCS#1-formatted ASN.1, this function can be used to pass in the raw
+    /// bytes for the public key components as `untrusted::Input` arguments.
+    ///
+    /// Takes an `RSAParameters` static to determine the parameters used,
+    /// a message, a signature (whose length should match the length of public
+    /// exponent, `n`) and a tuple containing the parts of the public key (in
+    /// big-endian byte order). The key length must be between 2048 and 8192 bits.
+    // Note that test coverage for this function thoroughly depends on test
+    // coverage for the `super::verify()` implementation. If we change that,
+    // test coverage for `verify_rsa()` will need to be reconsidered.
+    pub fn verify_rsa(params: &super::RSAParameters, msg: untrusted::Input,
```

Comment:
Update the commit message to refer to `ring::signature::primitive::verify_rsa` instead of `rsa::primitive::verify()`.